src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,97 @@
+import Text from "src/components/Text";
+import React, { useState } from "react";
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+
+import * as S from "./styles";
+
+const ProfileContent: React.FC<Record<string, never>> = () => {
```

Comment:
Can you try to explain what's the purpose of `Record` in TypeScript and what's the purpose here

Reply:
gw gatau si mon ternyata gw ilangin it works, so gw ilangin

Reply:
LOL can check record type in typescript, actually by default react fc takes {}, and the record itu purely nonsensical here, record itu kayak pair btw